unified-codes/libs/data/src/lib/v1/Entity.ts

```diff
@@ -2,7 +2,7 @@ import { IProperty, Property } from './Property';
 
 export enum EEntityCategory {
   DRUG = 'drug',
-  MEDICINAL_PRODUCT = 'medicinal_product',
+  CONSUMABLES = 'consumables',
```

Comment:
For consistency, may as well just call this category `CONSUMABLE` (can leave the label as `"Consumables"`)?

Reply:
Good idea! e9d8d341b